__tests__/tripRequestTest/tripRequestReducer.test.js

```diff
@@ -0,0 +1,32 @@
+import tripRequestReducer from '../../src/reducers/tripRequestReducer';
+describe('signIn reducer test ', () => {
+	it('should return success message', () => {
+		const action = { type: 'REQUEST_TRIP_SUCCESS' };
+		const initialState = {
+			error: '',
+			message: undefined,
+			tripCreated: undefined,
+		};
+		expect(tripRequestReducer({}, action)).toEqual(initialState);
+	});
+	it('should return error', () => {
+		const action = { type: 'REQUEST_TRIP_FAILURE' };
+		const initialState = {
+			error: undefined,
```

Comment:
@Jajabenit250 Even if you're mocking the expected results, it's good to use meaningful data. Since you're testing the presence of an error message, just don't define the error as undefined.